rkt/run.go

```diff
@@ -457,14 +457,14 @@ func (e *envFileMap) Set(s string) error {
 			continue
 		}
 		pair := strings.SplitN(scanner.Text(), "=", 2)
-		// Malformed lines
-		if len(pair) != 2 || len(pair[0]) == 0 {
-			return fmt.Errorf("environment variable must be specified as name=value (file %q)", file)
-		}
 		// Skip comments
 		if string(pair[0][0]) == "#" || string(pair[0][0]) == ";" {
 			continue
 		}
+		// Malformed lines
+		if len(pair) != 2 || len(pair[0]) == 0 {
```

Comment:
If we move this check here, we don't have any guarantee anymore that `pair[0][0]` actually exist. That array access above will panic on lines containing only a single `=` char.